website/install.rst

```diff
@@ -460,7 +460,7 @@ Component Options
   Install the binary programs (exrheader, exrinfo,
   exrmakepreview, etc). Default is ``ON``.
   
-* ``OPENEXR_INSTALL_EXAMPLES``
+* ``OPENEXR_BUILD_EXAMPLES``
 
   Build and install the example code. Default is ``ON``.
```

Comment:
Should remove any reference to install here:
```suggestion
  Build the example code. Default is ``ON``.
```

Reply:
Thank you! I've submitted an updated commit.